events/2020-10-10.interspeciesio-ml-workshop.md

```diff
@@ -1,25 +1,109 @@
-# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop hosted by Interspecies.IO
+# October 10, 2020 - Earth Species Project Machine Learning & Bioacoustics Workshop
 
-**TODO:** Copy describing the meeting
-- Time/Date (with link to https://savvytime.com/converter/pst-to-mst-est-cest or some such calculator)
-- less info in the title and more here?
-- Attendees? (greater consideration given the number of people we had in attendance)
+## hosted by Interspecies.IO
 
-## Watch
+It was a tremendous honor to present to the [Interspecies.io](https://interspecies.io/) community and share our latest work. The workshop was hosted by Interspecies.IO, a multidisciplinary group founded by Peter Gabriel (musician and activist), Dr. Vint Cerf (Vice President at Google and Co-designer of the Internet), Dr. Diana Reiss (Professor at Hunter College, a cognitive psychologist, and marine mammal scientist), and Dr. Neil Gershenfeld (head of MIT’s Center for Bits and Atoms).
 
+## Attendees: 
+
+Nearly 100 biologists, computational linguists, information theorists, musicians, and technologists. Continue the conversation with us [here](https://github.com/earthspecies/project/discussions/35). 
+
+
+## Recording: 
 [![video thumb](https://i.imgur.com/XwZq1TS.png)](https://archive.org/details/20201010-interspecies-i-o-earth-species-project-ai-toolbox)
 
-## Discuss
 
-https://github.com/earthspecies/project/discussions/35
+## Opensource Code: 
+ 
+ * [Cocktail Party Problem](https://github.com/earthspecies/cocktail-party-problem)
+ * [Acoustic representation toolbox ](https://github.com/earthspecies/acoustic-representation-toolbox)
+ * [Denoising, detection, and classification](https://github.com/earthspecies/dolphin-strandings)
+ * [ESP Library](https://github.com/earthspecies/library)
+
+
+## Participate: 
+
+The Earth Species Library is a collection of universally accessible bioacoustics datasets, ranging from humpback whales to zebra finches, tuned for machine learning. These datasets will enable machine learning researchers to train and validate novel machine learning methodologies and develop increasingly complex approaches to answer some of biology's most elusive questions. If you are interested in contributing your bioacoustic data and annotations to the Earth Species Library, please get in touch with [katie@earthspecies.org](mailto:katie@earthspecies.org). 
+
+## Getting Started: 
+
+*   [Introduction to using Collab](https://colab.research.google.com/notebooks/intro.ipynb)
+*   [Digital Signal Processing with Python](https://github.com/earthspecies/intro-to-DSP-with-python)
+
+## Resources: 
```

Comment:
I would roll getting started into resources.

Reply:
Yup! Looks good - maybe just 'Introduction to Colab' vs Collab